README.md

```diff
@@ -1,2 +1,11 @@
-# icd11
-ICD11 foundation ingest.
+# ICD11 Foundation Ingest
+
+## Pre-reqs
+- Docker
+- Docker images  
+  One or both of the following, depending on if you want to run the stable build `latest` or `dev`:
+  - a. `docker pull obolibrary/odkfull:latest`
+  - b. `docker pull obolibrary/odkfull:dev`
+
+## Running
+`sh run.sh make all`
```

Comment:
## Docs
Now I'm bundling other things that could be put into another PR. But after adding `run.sh`, I realized I needed docs for it, so I updated the readme with new sections:
- Pre- reqs
- Running

Reply:
Thanks, looks good.